File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/06.Nested-Loop/03-additional/10.profit.js
function profit(input) {

    let coinsOfOneLeva = Number(input[0]);
    let coinsOfTwoLeva = Number(input[1]);
    let fiveLeva = Number(input[2]);
    let sum = Number(input[3]);

    for (let i = 0; i <= coinsOfOneLeva; i++) {
        for (let j = 0; j <= coinsOfTwoLeva; j++) {
            for (let k = 0; k <= fiveLeva; k++) {
                if (i * 1 + j * 2 + k * 5 == sum) {
                    console.log(`${i} * 1 lv. + ${j} * 2 lv. + ${k} * 5 lv. = ${sum} lv.`);
                }
            }
        }
    }
}
profit(['5', '3', '1', '7']);

File: Python_Basics/Homeworks-And-Labs/06.nested_loops/03.additional/10.profit.py
coins_of_one_leva = int(input())
coins_of_two_leva = int(input())
five_leva = int(input())
sum = int(input())

for i in range(coins_of_one_leva + 1):
    for j in range(coins_of_two_leva + 1):
        for k in range(five_leva + 1):
            if i * 1 + j * 2 + k * 5 == sum:
                print(f'{i} * 1 lv. + {j} * 2 lv. + {k} * 5 lv. = {sum} lv.')
